doc/admin-guide/files/sni.yaml.en.rst

```diff
@@ -52,11 +52,34 @@ for a more detailed description of HTTP/2 connection coalescing.
 .. _override-host-sni-policy:
 .. _override-h2-properties:
 
+The following fields make up the key for each item in the configuration file.
+
 ========================= ========= ========================================================================================
 Key                       Direction Meaning
 ========================= ========= ========================================================================================
-fqdn                      Both      Fully Qualified Domain Name. This item is used if the SNI value matches this.
+fqdn                      Both      Fully Qualified Domain Name.
+
+inbound_port_range        Inbound   The port range for the inbound connection in the form ``port`` or
+                                    ``min:max``.
```

Comment:
Why does this use ';' and the example '-'? Don't use colon, it's already problematic in this context. '-' is the goto separator for a range.